frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -266,6 +268,15 @@ function InsightMeta({
                 </>
             }
             metaDetails={<InsightDetails insight={insight} />}
+            samplingNotice={
+                insight.filters.sampling_factor && insight.filters.sampling_factor < 1 ? (
```

Comment:
not blocking, we should have the tooltip text in the "show details" content when expanded. In case someone misses the icon, they'll see the text if they click show details

Reply:
Would be nice to get @lharries take on this - thought about putting this in 'Show details' as well but I think it's then not immediately clear to the user that the graph is sampled? Particularly to the users that didn't make the insight?

Reply:
(Assuming this is referring to the dashboard)

I think it would be good to make this a bit more obvious otherwise there's risk that they don't realize and think it's a data integrity issue, maybe like this? (Obviously not in red text however)
![Screenshot feat(sampling) initial sampling ui by yakkomajuri · Pull Request #14488 · PostHogposthog (Google Chrome) 2023-03-02 at 12 23@2x](https://user-images.githubusercontent.com/22766134/222427980-12fc9112-fb91-41ec-b2b0-b26967dcc8d5.png)

Reply:
Annoyingly that card is already so crowded. I initially had a notice just like that not so far from where you've put it. 

I'll play around with different options

Reply:
This was my proposal from before: 
<img width="856" alt="Screenshot Untitled – Figma (Google Chrome) 2023-03-02 at 14 04@2x" src="https://user-images.githubusercontent.com/22766134/222450713-3fbd55ed-1466-4dd5-a955-069ef23cb42f.png">

Reply:
I do like that. We certainly don't need to tell people what type of graph it is, nor show that it has no description